tasks/main.yml

```diff
@@ -411,49 +412,50 @@
     mode: "0644"
   register: configuration
 
-- name: create paperlessng venv
+- name: Create paperlessng venv
   become: true
   become_user: "{{ paperlessng_system_user }}"
   ansible.builtin.command:
     cmd: python3 -m venv {{ paperlessng_virtualenv }}
     creates: "{{ paperlessng_virtualenv }}"
   register: venv
 
-- name: install latest pip
+- name: Install latest pip/pipenv/whell
   become: true
   become_user: "{{ paperlessng_system_user }}"
   ansible.builtin.pip:
-    name: pip
+    name:
+    - pip
+    - wheel
+    - pipenv
     executable: "{{ paperlessng_virtualenv }}/bin/pip3"
     state: latest
   when: not reconfigure_only
 
 - name: Install needed requirements
   block:
-    - name: Replace pikepdf version if using ngx version lower than v1.8.0
-      ansible.builtin.lineinfile:
-        path: "{{ paperlessng_directory }}/requirements.txt"
-        regexp: '^pikepdf(.*)'
-        line: 'pikepdf==5.4.0'
-      when: not paperlessng_version == "v1.8.0" or
-            paperlessng_version == "latest"
-
-    - name: install paperlessng requirements
+    - name: Create requirments.txt file
+      ansible.builtin.shell: ".venv/bin/pipenv requirements > requirements.txt"
```

Comment:
Maybe this should check if the requirements.txt exists already?  An install from the release archive would, but a source install wouldn't?

Reply:
I'll add something in later on tonight